jetson/nucleo_bridge/LCMHandler.cpp

```diff
@@ -71,8 +68,8 @@ void LCMHandler::InternalHandler::carousel_openloop_cmd(LCM_INPUT, const Carouse
 
 void LCMHandler::InternalHandler::foot_openloop_cmd(LCM_INPUT, const FootCmd *msg)
 {
-    ControllerMap::controllers["FOOT_SCOOP"]->open_loop(msg->microscope_triad);
-    ControllerMap::controllers["FOOT_SENSOR"]->open_loop(msg->scoop);
+    ControllerMap::controllers["FOOT_SENSOR"]->open_loop(msg->microscope_triad);
```

Comment:
why....is this called the foot

Reply:
it was called the foot before so i just stuck to it... i guess we can change it to be ScienceHandCmd or something similar @CameronTressler , idk if that sounds that much better though

Reply:
i see...yeah ig